kuksa_databroker/databroker/src/viss/v2/server.rs

```diff
@@ -253,7 +253,15 @@ impl Viss for Server {
         let Some(entries) = broker
             .get_id_by_path(request.path.as_ref())
             .await
-            .map(|id| HashMap::from([(id, HashSet::from([broker::Field::Datapoint]))]))
+            .map(|id| {
+                HashMap::from([(
+                    id,
+                    (
+                        HashSet::from([broker::Field::Datapoint]),
+                        ChangeType::Static,
+                    ),
+                )])
+            })
```

Comment:
I don't see how it makes sense to add the `ChangeType` argument to the subscribe function as that is a property already associated with each signal (see other comment). And if it did make sense, why set it to `ChangeType::Static` here?

Reply:
Changes reverted